cs/v1/CS_v1.0.0.yaml

```diff
@@ -556,85 +675,46 @@ components:
           type: string
           format: date-time
           example: '2025-01-14T09:21:00+01:00'
-          x-stoplight:
-            id: tfufl59x6te0n
           description: 'The local date and time, when the event will take place, in ISO 8601 format specifying the offset.'
     PointToPoint:
       title: Point To Point
-      x-stoplight:
-        id: x1x7ae0q7kym3
       type: object
       required:
         - placeOfReceipt
         - placeOfDelivery
         - legs
+        - legsx
       properties:
         placeOfReceipt:
-          $ref: '#/components/schemas/Place'
+          $ref: '#/components/schemas/PlaceOfReceipt'
         placeOfDelivery:
-          $ref: '#/components/schemas/Place'
+          $ref: '#/components/schemas/PlaceOfDelivery'
         cutOffTimes:
           type: array
           description: A list of cut-offs times provided by the carrier when available. A cut-off time indicates the latest deadline within which a task must be completed.
-          x-stoplight:
-            id: ey85uy7hk8gp5
           items:
             $ref: '#/components/schemas/CutOffTime'
         transitTime:
           type: integer
-          x-stoplight:
-            id: n94kpnvpk6zsn
           description: The estimated total time in **days** that it takes a shipment to move from place of receipt to place of delivery. Transit time includes stop-over time during transshipments and waiting time at connection points.
           format: int64
           example: 10
         legs:
-          x-stoplight:
-            id: w4k1wtp4z5j6i
           type: array
           minItems: 1
           items:
             $ref: '#/components/schemas/Leg'
-    Leg:
-      title: Leg
-      x-stoplight:
-        id: itbpqbyi0so58
-      type: object
-      description: Leg of the Point-to-Point routing. The object "Legs" can be conformed by as many legs as needed and this legs must be identified with a sequence number.
-      required:
-        - departure
-        - arrival
-      properties:
-        sequenceNumber:
-          type: integer
-          x-stoplight:
-            id: 4z9lhffp3cyjs
-          format: int64
-          description: Sequence number of the leg.
-        modeOfTransport:
-          x-stoplight:
-            id: 9atywo7bqza4b
-          description: The mode of transport as defined by DCSA.
-          oneOf:
-            - $ref: '#/components/schemas/VesselTransport'
-            - $ref: '#/components/schemas/BargeTransport'
-            - $ref: '#/components/schemas/RailTransport'
-            - $ref: '#/components/schemas/TruckTransport'
-            - $ref: '#/components/schemas/CombinedTransport'
-        departure:
-          $ref: '#/components/schemas/Place'
-        arrival:
-          $ref: '#/components/schemas/Place'
+        legsx:
+          type: array
+          items:
```

Comment:
`legsx` is a required field - I suggest to add a `minItems:1` so you don't forget later...